pages/about.tsx

```diff
@@ -234,8 +236,235 @@ export default function About({ pageProps }) {
             <WorkingAtGrouparooCard />
           </Col>
         </Row>
+
+        <br />
+        <h2>Working at Grouparoo</h2>
+
+        <Image
+          src="/images/team/about-team.png"
+          alt="The Grouparoo Team"
+          height="720"
+          width="3200"
+        />
+
```

Comment:
```suggestion
<br />
```
Just a bit of breathing room